help/de/Help_ch_GUIMenus.xml

```diff
@@ -1907,7 +1899,8 @@
 
         <table frame="topbot" id="AccTree-ReportsMenu">
           <title>Kontenhierarchie - Berichte-Menu - Zugriff auf &app; Berichte und Diagramme</title>
-
+<!-- Translators: Attention, the menu items in GnuCash differ in the order between C and de.
+                  The actual sequence from the menu "Reports" is shown below. -->
```

Comment:
Perhaps we should at least add id's to the rows of the main entries?

Reply:
@Bob-IT, @fellen: Do you have any idea why the order of the menu items in the reports differs in the translation into German compared to the original?

Reply:
AFAIK if nothing else is set, the sorting is alphabetical by the labels of  the menu entries.
Perhaps related: https://developer.gnome.org/gtk3/stable/GtkTreeModelSort.html